src/components/Form/Fields.js

```diff
@@ -62,7 +62,7 @@ export const DataSelect = ({ request, optionsMap, name, ...props }) => {
     });
 
     const renderOption = useCallback(
-        ({ label }) => <span data-cy={`${name}-${label}`}>{label}</span>,
+        ({label, value}) => <span data-cy={`${name}-${value}`}>{label}</span>,
```

Comment:
Is there a reason for doing this by option value instead of display label?

Reply:
this is for testing purposes - it's difficult to select items by label but values, you can always assume it's by an id that starts from 1, so it's easier to select that.